src/smt_util/node_traversal.cpp

```diff
@@ -0,0 +1,101 @@
+#include "node_traversal.h"
```

Comment:
Add our usual header with copyright info.

Reply:
Done in b3eb1e2bf